utils/wasm-gen/src/config/syscalls/param.rs

```diff
@@ -55,7 +55,7 @@ impl Default for SysCallsParamsConfig {
                 (ParamType::Gas, (0..=250_000_000_000).into()),
                 (ParamType::MessagePosition, (0..=10).into()),
                 (ParamType::Duration, (0..=100).into()),
-                (ParamType::Delay, (0..=100).into()),
+                (ParamType::Delay, (1..=4).into()),
```

Comment:
why start from 1?

Reply:
Zero duration is not allowed, isn't it?

Reply:
> Zero duration is not allowed, isn't it?

It's delay, not duration. Duration exactly can't be started from 0

Reply:
done